proton

```diff
@@ -924,6 +924,9 @@ class Session:
         self.check_environment("PROTON_HEAP_DELAY_FREE", "heapdelayfree")
         self.check_environment("PROTON_ENABLE_NVAPI", "enablenvapi")
 
+        # This layer is horrifically broken.
+        self.env["DISABLE_LAYER_AMD_SWITCHABLE_GRAPHICS_1"] = "1"
```

Comment:
Looking a bit further up in this file, would it be better to do?
`self.env.setdefault("DISABLE_LAYER_AMD_SWITCHABLE_GRAPHICS_1", "1")`
I think that would allow for easier retesting later? (untested)

Reply:
Done.